src/v2/wallet.js

```diff
@@ -1151,6 +1151,18 @@ Wallet.prototype.prebuildAndSignTransaction = function(params, callback) {
   }).call(this).asCallback(callback);
 };
 
+/**
+ * Accelerate pending transaction by increasing the fee rate. (Beta)
+ *
+ * Currently only implemented for Bitcoin as Child-Pays-For-Parent (CPFP).
```

Comment:
I think what's meant is that since platform and SDK are supposed to be coin agnostic, the accelerateTx call could resolve to a different function for another coin eventually.

Reply:
I see. Marking as resolved.